eq_translations/schema_translation.py

```diff
@@ -18,22 +18,28 @@ def save(self, translation_file_path):
         with open(translation_file_path, 'w+b') as translation_file:
             pofile.write_po(translation_file, self.catalog)
 
+    @staticmethod
+    def dumb_strings_equal(message_a, message_b):
+        return dumb_to_smart_quotes(message_a) == dumb_to_smart_quotes(message_b)
+
+    @staticmethod
+    def get_comment_answer_ids(comments):
+        return [comment.split(':')[1].strip() for comment in comments if 'answer-id' in comment]
+
     def translate_message(self, message_to_translate, answer_id=None, message_context=None):
         for message in self.catalog:
-            if message.id and dumb_to_smart_quotes(message.id) == dumb_to_smart_quotes(message_to_translate):
+            if message.id and SchemaTranslation.dumb_strings_equal(message.id, message_to_translate):
```

Comment:
This static method would make more sense in the utils module as a function alongside the `dumb_to_smart_quotes` function